File: app/src/main/java/ru/ushell/app/utils/SavingSession/Session.java
package ru.ushell.app.utils.SavingSession;

import android.content.Context;
import android.content.SharedPreferences;

public class Session {

    public static void save(Context context, String value, String key) {
        SharedPreferences sharedPreferences = context.getSharedPreferences("clipcode", Context.MODE_PRIVATE);
        SharedPreferences.Editor editor = sharedPreferences.edit();
        editor.putString(key, value);
        editor.apply();

    }

    protected static String get(Context context, String key) {
        SharedPreferences sharedPreferences = context.getSharedPreferences("clipcode", Context.MODE_PRIVATE);
        return sharedPreferences.getString(key, "false");
    }
}

File: app/src/main/java/ru/ushell/app/domain/service/session/Session.kt
package ru.ushell.app.domain.service.session

import android.content.Context

class Session {
    companion object {

        fun isLogin(context: Context): Boolean =
            SessionRepository.get(context, SessionStaticVar.KEYLOGGER).toBoolean()

        fun setLogin(context: Context) =
            SessionRepository.save(context, "true", SessionStaticVar.KEYLOGGER)

        fun userLogout(context: Context) =
            SessionRepository.save(context, "false", SessionStaticVar.KEYLOGGER)
    }
}